tests/javascripts/sharing-button.test.js

```diff
@@ -0,0 +1,195 @@
+import helpers from './support/helpers.js'
+import sharingButton from '../../src/assets/javascripts/sharing-button.mjs'
+
+const URL = 'https://www.gov.uk/alerts/3-may-2021'
+const htmlFixture = `
+      <div class="share-url">
+        <p class="govuk-body">
+          ${URL}
+        </p>
+      </div>`
+
+afterAll(() => {
+  // clear up methods in the global space
+  document.queryCommandSupported = undefined
```

Comment:
Also, could we do `delete` instead of `= undefined` (similarly for other assignments below)? My assumption is that `delete` is more realistic, and in any case it's helpful if we can be consistent.

Reply:
Sure, I borrowed this line from [a similar test in the admin app](https://github.com/alphagov/notifications-admin/blob/2871d4cb0433964b2e96f07644aee314b6d576c3/tests/javascripts/apiKey.test.js#L8) where, I think, `delete` isn't used elsewhere so there isn't the same requirement for consistency. The form including an assignment is something I picked up a while ago, which I think is intended to be less ambiguous than `delete` (because, in JavaScript you can often be wrong about what an operation will perform :-S). Happy to convert to use `delete` here for consistency (done in https://github.com/alphagov/notifications-govuk-alerts/pull/105/commits/f1f3493e607847e8d7b738af605c811eacbe1415).

Reply:
Oh, interesting: I would say `delete` is less ambiguous than `undefined`. Aside from consistency, using `delete` means the object is restored back to what it was, whereas `= undefined` means it still has the property declared on it. I doubt it matters much either way, so just being consistent in the same file is fine with me.